js/main.js

```diff
@@ -0,0 +1,95 @@
+const libraryDiv = document.getElementById('Library');
+const errorElem = document.getElementById('errors');
+const btnNewBook = document.getElementById('btnNewBook');
+const btnAddBook = document.getElementById('btnAddBook');
+let myLibrary = [];
+
+function Book(title, author, desc) {
+  this.constructor = Book;
+  this.id = Math.random().toString(36).substr(2, 9);
+  this.title = title;
+  this.author = author;
+  this.desc = desc;
+  this.wasRead = false;
+}
+
+Book.prototype.toggleRead = function toogleRead() {
+  this.wasRead = !this.wasRead;
+};
+
+/* eslint-disable no-use-before-define */
+const removeBook = (e) => {
+  const { bookId } = e.target.dataset;
+  myLibrary = myLibrary.filter(book => book.id !== bookId);
+  localStorage.setItem('myLibrary', JSON.stringify(myLibrary));
+  render();
+};
+
+const readBook = (e) => {
+  const { bookId } = e.target.dataset;
+  const book = myLibrary.find(book => book.id === bookId);
+  book.toggleRead();
+  localStorage.setItem('myLibrary', JSON.stringify(myLibrary));
+  render();
+};
+/* eslint-enable no-use-before-define */
+
+const bookToHTML = (book) => `
```

Comment:
Let separate the DOM from application logic
![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/